### DISCUSSION TITLE
dang not open source

### DISCUSSION THREAD

**Discussion by icodeintx** [thunderclient/thunder-client-support#864] [General]
I was hoping to participate to make enhancements.  Love this extension so much I dropped all those other 'clients' 

Keep up the great work.

**Comment by rangav** [comment]
@icodeintx glad you like the extension, have plans to open source some modules at later stage.

**Comment by icodeintx** [comment]
I would love to see extended import/export.  As I get my friends to use Thunder Client and we are working on the same project it would be nice to have options to send (export) request, collections, all, depending on what I select.

I would also like to see a configuration option that lets me choose where all of my info is stored for the sake of backups.  A few times I have gone on a mission to find the config and have found it but what a task.  🤔

Great extension!

**Comment by rangav** [comment]
Hi @icodeintx 

Did not understood your question about export, can you provide clear examples

request files are stored here
https://github.com/rangav/thunder-client-support#storage

you can also save to git
https://github.com/rangav/thunder-client-support#team
